pkg/rtc/transportmanager.go

```diff
@@ -609,3 +623,32 @@ func (t *TransportManager) ProcessPendingPublisherDataChannels() {
 		}
 	}
 }
+
+func (t *TransportManager) OnReceiverReport(dt *sfu.DownTrack, report *rtcp.ReceiverReport) {
+	t.mediaLossProxy.HandleMaxLossFeedback(dt, report)
+}
+
+func (t *TransportManager) onMediaLossUpdate(loss uint8) {
+	t.udpLossUnstableCount <<= 1
```

Comment:
This needs to be locked? A lot of tracks could be flowing through this I guess.

Reply:
The MediaLossProxy has a minimum update interval (1 sec), so it is safe without lock

Reply:
Yeah. But, it is probably okay to lock as it is once a second. Just don't want code to behave unexpectedly if something else changes. So, if it is not critical path, have been advocating safety :-)

Reply:
Done